src/crimson/os/cyan_collection.h

```diff
@@ -24,12 +25,8 @@ class Object;
  * ObjectStore users my get collection handles with open_collection() (or,
  * for bootstrapping a new collection, create_new_collection()).
  */
-struct Collection : public boost::intrusive_ref_counter<
-  Collection,
-  boost::thread_unsafe_counter>
-{
+struct Collection : public ObjectStore::CollectionImpl {
```

Comment:
what's the merit of using `ObjectStore::CollectionImpl` to access `Collection` ?

Reply:
I just use the existing implementation.  that ObjectStore::CollectionImpl as parent class will not expose collection details defined by each store to osd other code.